docs/new_flux_ideas_and_guide.md

```diff
@@ -0,0 +1,985 @@
+bottom are examples of selectors. Some selectors also double as aggregates like min and max.
+
+This is an important distinction. An aggregate produces new data from some source data. That new data is frequently a 
+summary statistic that describes the source data in some meaningful way. A selector specifies criteria for selecting 
+rows from some source data. So you're not transforming the data when using a selector. For example, `count` is not a 
+function that could be used as a selector.
+
+#### Sample Dataset
+Here's an example dataset that we'll use for the rest of the examples in this section. The raw number timestamps 
+have been replaced with human readable times and the values have been set to simple whole numbers to make things 
+easier. The example times only go down to minute precision to make things easier, but you can summarize down to 
+the nanosecond.
+
+```
+h2o_temperature,location=santa_monica,state=CA surface_degrees=65,bottom_degrees=50 2020-02-22T15:01
+h2o_temperature,location=santa_monica,state=CA surface_degrees=64,bottom_degrees=49 2020-02-22T15:31
+h2o_temperature,location=santa_monica,state=CA surface_degrees=63,bottom_degrees=49 2020-02-22T16:01
+h2o_temperature,location=santa_monica,state=CA surface_degrees=62,bottom_degrees=49 2020-02-22T16:31
+h2o_temperature,location=santa_monica,state=CA surface_degrees=61,bottom_degrees=48 2020-02-22T17:01
+h2o_temperature,location=santa_monica,state=CA surface_degrees=60,bottom_degrees=48 2020-02-22T17:31
+h2o_temperature,location=santa_monica,state=CA surface_degrees=60,bottom_degrees=48 2020-02-22T17:46
+h2o_temperature,location=coyote_creek,state=CA surface_degrees=55,bottom_degrees=51 2020-02-22T16:05
+h2o_temperature,location=coyote_creek,state=CA surface_degrees=53,bottom_degrees=50 2020-02-22T17:05
+h2o_temperature,location=puget_sound,state=WA surface_degrees=55,bottom_degrees=40 2020-02-22T17:10
+h2o_temperature,location=puget_sound,state=WA surface_degrees=54,bottom_degrees=40 2020-02-22T17:40
+```
+
+In the data set we can see there are three distinct keys and that the time range of data is from 
+`2020-02-22T15:01` to `2020-02-22T17:46` (about a two and a half hour span). There are more data 
+points for the `santa_monica`, `CA` location than for the others. The differing number of values 
+will be instructive later when we shape and summarize this data.
+
+#### Summarizing and Grouping with Aggregates
+Let's get into an example for summarizing some of our water data. For the sake of these queries, assume that 
+we're executing them at `2020-02-22T18:00`, that way we can use relative times in the query. Say we want to get the 
+last 3 hours of data, with the min, max, and mean of `bottom_degrees` for that time period. With that 
+definition, we'd expect to get 1 row per series in the dataset (assuming that we have min, max, and mean as 
+columns in the resulting table. Here's the query:
+
+```
+@my_data.h2o_temperature{bottom_degrees, time > -3h}
```

Comment:
what happens if i don't add `bottom_degrees` here? My guess is that it would return all columns and bottom_degress, min/max/mean_bottom_degrees would all the same value?